### DISCUSSION TITLE
Easy way to view `tar_meta()` attributes of most recently updated targets?

### DISCUSSION THREAD

**Discussion by ajjitn** [ropensci/targets#402] [Help] [answered]
Hey everyone! Just started using `targets` and I'm already a massive fan! 

I often want to know how long the targets took to build in the most recent run of my pipeline, but just the targets that were updated in the most recent run. So if just the last two objects in the pipeline were updated in the last run, I would like to know how long each target took to build. Is there an easy way to get a `run_id` or `run_timestamp` field in the dataframe returned by `tar_meta()` that makes this easy?

**Comment by wlandau** [comment, marked as answer]
`tar_progress()` shows which targets were not skipped in the last run of the pipeline. You can use the result to filter the output of `tar_meta()`.

**Reply by ajjitn** [reply]
Nifty! Thanks Will
